File: src/main/java/br/com/rfasioli/micrometermetrics/resource/dto/PersonDTO.java
package br.com.rfasioli.micrometermetrics.resource.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

@Data
@Builder
@AllArgsConstructor
public class PersonDTO {
  private String id;
  private String firstName;
  private String sureName;
  private String lastName;
}

File: src/main/kotlin/br/com/rfasioli/micrometermetrics/resource/dto/PersonDTO.kt
package br.com.rfasioli.micrometermetrics.resource.dto

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Data

data class PersonDTO (
    val id: String? = null,
    val firstName: String,
    val sureName: String? = null,
    val lastName: String
)
